src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp

```diff
@@ -342,7 +352,14 @@ class OpenGLDrawingEngine : public IDrawingEngine
     void PaintRain() override
```

Comment:
```suggestion
    void PaintWeather() override
```
This is no longer only responsible for rain.